sig-apps/Charter.md

```diff
@@ -0,0 +1,141 @@
+# SIG Apps Charter
+
+## Scope
+
+SIG Apps Covers deploying and operating applications in Kubernetes with a focus on the application developer and application operator experience.
+
+For example, this includes:
+
+* Discussions on all levels from application development, application testing environments, and applications running in production
+* Discussion of how to define and run apps in Kubernetes
+* Demos of relevant tools and projects
+* Discussion on areas of friction that can lead to suggesting improvements or feature requests
+* Development of Kubernetes primitives (e.g., Workloads API) to enable the application developers and operators. Note, most new types should be created as CRDs during their development phase and may always be ecosystem projects
+* Creation of documentation to enable application developers and operators
+* Development and maintenance of the sub-projects of Helm, Charts, and Kompose. Note, these projects are grandfathered and treated as part of the ecosystem but owned by SIG Apps. New projects are expected to stay in the ecosystem.
+
+### Goals
+
+* Discuss running and defining applications in Kubernetes (e.g., APIs, SDKs, Controllers, package management tools, etc.)
+* Share common patterns and practices
+* Work on improvements to the Workload API
+* Suggest and sometimes develop Kubernetes features where we see friction
+* Be the voice of the people running applications in Kubernetes (application developers and operators)
+* Help people get involved in the Kubernetes community
+* Show early features/demos of tools that make running apps easier
```

Comment:
Showing demos doesn't seem like a goal. It's an implementation detail that doesn't need to be covered in the charter. It's fine to just mention it in the SIG Apps meeting agenda/notes doc.